libvuln/libvuln.go

```diff
@@ -57,13 +59,17 @@ func New(ctx context.Context, opts *Opts) (*Libvuln, error) {
 		pool:     pool,
 		matchers: opts.Matchers,
 	}
+	l.UpdateDriver, err = NewUpdater(pool, opts.Client, opts.UpdaterConfigs, opts.UpdateWorkers, opts.UpdaterFilter)
+	if err != nil {
+		return nil, err
+	}
 
 	// Run updaters synchronously, initially.
-	if err := l.RunUpdaters(ctx, opts.UpdateWorkers, setFuncs...); err != nil {
+	if err := l.RunUpdaters(ctx, setFuncs...); err != nil {
 		return nil, err
 	}
 	if !opts.DisableBackgoundUpdates {
```

Comment:
Previously, you would disable background updaters by specifying an empty list in the Udpater's config. Is an bool desired over that concept? If so have we made sure that the previous logic does not interfere with this logic? For example if this bool is set but the list is not-empty does the correct thing happen?